apps/vr-tests/src/stories/DetailsHeader.stories.tsx

```diff
@@ -0,0 +1,185 @@
+const getClassNames = classNamesFunction<IDetailsHeaderStyleProps, IDetailsHeaderStyles>();
+
+// const _onDragStart = (item?: any, itemIndex?: number, selectedItems?: any[], event?: MouseEvent): void => {
+
+//   if (itemIndex) {
+//     this._updateHeaderDragInfo(itemIndex);
+//     this._root.current.classList.add(classNames.borderWhileDragging);
+//   }
+// }
+
+// const updateDragInfo = (props: { itemIndex: number }, event?: MouseEvent) => {
+
+//     this._draggedColumnIndex = this.props.selectionMode !== SelectionMode.none ? itemIndex - 2 : itemIndex - 1;
+//     onColumnDragStart(true);
+
+// }
+
+// _classNames = getClassNames(styles);
+
+
+let _dragDropHelper: IDragDropHelper;
+
+const columns: IColumn[] = [
+  { key: 'a', name: 'a', fieldName: 'a', minWidth: 200, maxWidth: 400, calculatedWidth: 200, isResizable: true },
+  {
+    key: 'b',
+    name: 'b',
+    fieldName: 'b',
+    minWidth: 200,
+    maxWidth: 400,
+    calculatedWidth: 200,
+    isResizable: true,
+    isSorted: true,
+    sortAscendingAriaLabel: 'Sorted up.',
+    sortDescendingAriaLabel: 'Sorted down.',
+    ariaLabel: 'Click to sort.'
+  },
+  {
+    key: 'c',
+    name: 'c',
+    fieldName: 'c',
+    minWidth: 200,
+    maxWidth: 400,
+    calculatedWidth: 200,
+    isResizable: true,
+    columnActionsMode: ColumnActionsMode.hasDropdown,
+    isIconOnly: false,
+    isFiltered: true,
+    filterAriaLabel: 'Filtered.',
+    isGrouped: true,
+    groupAriaLabel: 'Grouped.',
+    ariaLabel: 'Click to sort, filter, or group.'
+  },
+  {
+    key: 'd',
+    name: 'd',
+    fieldName: 'd',
+    minWidth: 200,
+    maxWidth: 400,
+    calculatedWidth: 200,
+    isResizable: true,
+    columnActionsMode: ColumnActionsMode.hasDropdown,
+    isIconOnly: false,
+    isFiltered: true,
+    filterAriaLabel: 'Filtered.',
+    isGrouped: true,
+    groupAriaLabel: 'Grouped.',
+    ariaLabel: 'Click to sort, filter, or group.'
+  },
+  {
+    key: 'e',
+    name: 'e',
+    fieldName: 'e',
+    minWidth: 200,
+    maxWidth: 400,
+    calculatedWidth: 200,
+    isResizable: true,
+    columnActionsMode: ColumnActionsMode.hasDropdown,
+    isIconOnly: false,
+    isFiltered: true,
+    filterAriaLabel: 'Filtered.',
+    isGrouped: true,
+    groupAriaLabel: 'Grouped.',
+    ariaLabel: 'Click to sort, filter, or group.'
+  },
+  {
+    key: 'f',
+    name: 'f',
+    fieldName: 'f',
+    minWidth: 200,
+    maxWidth: 400,
+    calculatedWidth: 200,
+    isResizable: true,
+    columnActionsMode: ColumnActionsMode.hasDropdown,
+    isIconOnly: false,
+    isFiltered: true,
+    filterAriaLabel: 'Filtered.',
+    isGrouped: true,
+    groupAriaLabel: 'Grouped.',
+    ariaLabel: 'Click to sort, filter, or group.'
+  }
+];
+
+
+_selection.setItems(_items);
+
+
+const _columnReorderProps = {
+  frozenColumnCountFromStart: 1,
+  frozenColumnCountFromEnd: 1,
+  handleColumnReorder: this._dummyFunction
+};
+
+
+
+storiesOf('DetailsHeader', module)
```

Comment:
Does this storiesOf string need to be unique? maybe it's the 'add' function. But wanted to make sure this wasn't causing any problems.